backend/pipeline/serializers/crud.py

```diff
@@ -24,6 +26,34 @@ class Meta:
         model = Pipeline
         fields = "__all__"
 
+    def validate_cron_string(self, value: Optional[str] = None) -> Optional[str]:
```

Comment:
@muhammad-ali-e this function isn't used anywhere - can it be removed?

Reply:
It's Serializer validation, It will internally called by serializer.

Reply:
If possible add a docstring mentioning this - we wouldn't want to remove such functions stating that there's no usages